samples/client/petstore/powershell_test/src/IO.Swagger/IO.Swagger.psm1

```diff
@@ -0,0 +1,94 @@
+#region Import functions
+
+$FunctionsToExport = @()
+$Folders = 'Public', 'Private'
+
+foreach ($Scope in $Folders) {
+    Get-ChildItem -LiteralPath (
+        Join-Path -Path $PSScriptRoot -ChildPath $Scope
+    ) -File -Filter '*.ps1' | ForEach-Object {
+        $File = $_
+        try {
+            Write-Verbose "Dotsourcing file: $File"
+            . $File.FullName
+
+            switch ($Scope) {
+                'Public' {
+                    $FunctionsToExport += $File.BaseName
+                }
+            }
+        } catch {
+            throw "Can't import functions from file: $File"
+        }
+    }
+}
+
+Export-ModuleMember -Function $FunctionsToExport
+
+#endregion
+
+
+#region Initialize APIs
+
+'Creating object: IO.Swagger.Api.PetApi' | Write-Verbose
+$Script:PetApi= New-Object -TypeName IO.Swagger.Api.PetApi -ArgumentList @($null)
+
+'Creating object: IO.Swagger.Api.PetApi' | Write-Verbose
```

Comment:
Look like something gone wrong with mustache template, so it repeats the same line.